public/keymaps/a/annepro2_c15_default.json

```
@@ -0,0 +1,29 @@
+{
+  "keyboard": "annepro2/c15",
+  "keymap": "default",
+  "version": 1,
+  "layout": "LAYOUT_60_ansi",
+  "layers": [
+    [
+      "KC_GESC", "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS",        "KC_EQL",         "KC_BSPC",
```

I like these keymaps to match as closely as possible to their `qmk/qmk_firmware` equivalents.

```suggestion
      "KC_ESC",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS",        "KC_EQL",         "KC_BSPC",
```